src/Squisher.js

```diff
@@ -114,13 +114,53 @@ class Squisher {
         }
     }
 
-    updateHelper(node, playerFrames, whitelist) {
+    updateHelper(node, playerFrames, whitelist, scale) {
+        if (this.game.getRoot() === node) {
+            scale = {
+                x: (100 - config.BEZEL_SIZE.x) / 100,
+                y: (100 - config.BEZEL_SIZE.y) / 100
+            }
+        }
+
         if (!this.ids.has(node.node.id)) {
             this.ids.add(node.node.id);
             node.addListener(this);
         }
 
-        const squished = squish(node.node);
+        let squished;
+        if (scale) {
+            const scaledNode = unsquish(squish(node.node));
+
+            if (scaledNode.text) {
+                scaledNode.text.x *= scale.x;
+                scaledNode.text.x += (1 - scale.x) * 100 / 2;
```

Comment:
I don't quite understand why this calculation seems to work, I think it would help to just make it one line instead of two lines like so:

```js
scaledNode.text.x = scale.x * ( scaledNode.text.x - 50) + 50
```

you do a similar calculation in throughout this file as well, and should we should be able to blanket replace it with this version instead